pkgs/applications/altcoins/hivemind.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoreconfHook, openssl, db48, boost
+, zlib, miniupnpc, qt4, utillinux, protobuf, qrencode, libevent
+, withGui }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "hivemind" + (toString (optional (!withGui) "d")) + "-" + version;
+  version = "git-2017-04-13";
```

Comment:
> We use `unstable` as a suffix instead of `git` per convention to denote unreleased versions.

Thanks, fixed.

> Is their a good reason not to use a released version?

Just that the full network hasn't launched yet, so at the moment this package is only useful to testers and developers.